docs[patch]: In conceptual docs explain constraints on ToolMessage (#26792)
langchain-ai/langchain: docs/docs/concepts.mdx

- a `tool_call_id` field which conveys the id of the call to the tool that was called to produce this result.
- an `artifact` field which can be used to pass along arbitrary artifacts of the tool execution which are useful to track but which should not be sent to the model.

> With most chat models, a `ToolMessage` can appear in the conversation history following an `AIMessage` with tha requests one or more tool calls (via `tool_calls` field).

Author: ```suggestion
With most chat models, a `ToolMessage` can only appear in the chat history after an `AIMessage` that has a populated `tool_calls` field.
```
